src/compose/ComposeMenu.js

```diff
@@ -234,56 +227,58 @@ class ComposeMenuInner extends PureComponent<Props> {
   });
 
   render() {
-    const { expanded, insertVideoCallLink, onExpandContract } = this.props;
-    const numIcons =
-      2 + (Platform.OS === 'android' ? 1 : 0) + (insertVideoCallLink !== null ? 1 : 0);
+    const { expanded, insertVideoCallLink } = this.props;
+
+    const onOpenActionSheet = () => {
+      const options = [
+        'Upload a file',
```

Comment:
Please set up these new user-facing strings for translation. They should get entries in static/translations/messages_en.json. Please see https://github.com/zulip/zulip-mobile/blob/main/docs/howto/translations.md and search for `const _ = this.context;` in the codebase.